pkg/server/api.go

```diff
@@ -133,7 +133,7 @@ func (sh *APIHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		w.WriteHeader(http.StatusNotFound)
 		return
 	}
-	// we know we're at 3.2 in code.. serve directly, parsing is expensive...
+	// we know we're at 3.4 in code.. serve directly, parsing is expensive...
```

Comment:
nit: This comment seems to imply that the `if reqConfigVer.Equal(*semver.New("3.4.0"))` condition should now just return servedConf. I assume the conversion is really a no-op regardless, but just for cleanup, maybe we can just delete this line (or remove the conversion code if you want)

Reply:
Good catch. It's a no-op, but the *right* thing to do is take out the conversion, and that is what I have done :smile: